src/Core/System/Resources/config/listing.xml

```diff
@@ -38,6 +38,15 @@
             <helpText lang="de-DE">Aktivieren um Bewertungen anzuzeigen</helpText>
         </input-field>
 
+        <input-field type="bool">
+            <name>enableStockManagement</name>
+            <label>Enable stock management</label>
+            <label lang="de-DE">Bestandsverwaltung ermöglichen</label>
+            <helpText>Deactivate to disable stock management. Please note this only applies if you have the STOCK_HANDLING feature flag enabled. When disabled, none of the event subscribers for order transitions will be executed. In practice, this means that none of the subscribers in Shopware\Core\Content\Product\Stock\OrderStockSubscriber will be executed</helpText>
```

Comment:
I would add an hint, that this config is system-wide and cannot be toggled by sales channel. The UI will allow changing it per sales channel but we don't care while reading (and also cannot do that)